.github/workflows/build.yml

```diff
@@ -30,7 +31,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: 3.12
+          python-version: '3.12'
```

Comment:
@amotl why are we using 3.12 here and in the `.readthedocs.yml` below instead of 3.13?

Reply:
RTD apparently does not offer Python 3.13, yet.